code/index.html

```diff
@@ -1,30 +1,140 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="apple-touch-icon" sizes="180x180" href="./images/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./images/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
+    <title>Camden Road Studios</title>
+    <link rel="stylesheet" type="text/css" href="./style.css">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header class="header">
+      <h1 class="header-text">Camden Road Studios</h1>
+      <h2>write - record - release</h2> 
+      <div class="video-container">
+        <video autoplay loop muted>
+            <source src="./videos/video1.mp4" type="video/mp4">
+        </video>
+      </div>
+      <picture>
+        <source srcset="./images/record-img-small.jpg" media="(max-width:600px)" />
+        <source srcset="./images/record-img-med.jpg" media="(max-width:900px)" />
+        <img srcset="./images/record-img-med.jpg" alt="Image of a record playing" />
+      </picture>
+    </header>
+      
+    <section class="main-container">
+      <h3>Sign up and book a session</h3>
+        <div class="flex-box">
+      <form 
+          class="form" 
+          action="http://httpbin.org/anything" 
+          method="POST"
+      >
+        <div class="form-group">
+          <label for="fullName">Name</label>
+          <input 
+              type="text" 
+              id="fullName" 
+              placeholder="Full name" 
+              name="fullNameValue"
+              minlength="4" 
```

Comment:
It's great that you're using minlength as a type of validation too, but consider having 2 or 3 instead since there are people with names like Eva or Lo, for example 😎 But now I'm nitpicking!